src/installer/corehost/cli/bundle/runner.cpp

```diff
@@ -51,7 +51,7 @@ const file_entry_t*  runner_t::probe(const pal::string_t &relative_path) const
 {
     for (const file_entry_t& entry : m_manifest.files)
     {
-        if (pal::pathcmp(entry.relative_path(), relative_path) == 0)
+        if (entry.matches(relative_path))
```

Comment:
Can you please add at least asserts into the other 'probe' and 'locate' methods that when they return an entry the entry is never disabled? Just to double check.